packages/sdk/src/encryptedContentTypes.ts

```diff
@@ -13,6 +13,23 @@ export function isEncryptedContentKind(kind: string): kind is EncryptedContent['
     return kind === 'text' || kind === 'channelMessage' || kind === 'channelProperties'
 }
 
+// Type guard function based on field checks
+export function isEncryptedData(obj: unknown): obj is EncryptedData {
+    if (typeof obj !== 'object' || obj === null) {
+        return false
+    }
+
+    const data = obj as EncryptedData
+    return (
+        typeof data.ciphertext === 'string' &&
+        typeof data.algorithm === 'string' &&
+        typeof data.senderKey === 'string' &&
+        typeof data.sessionId === 'string' &&
+        (typeof data.checksum === 'string' || data.checksum === undefined) &&
+        (typeof data.refEventId === 'string' || data.refEventId === undefined)
+    )
+}
+
```

Comment:
looks like `isEncryptedData` is only for testing can we move it to a test utils file?

Reply:
moved